README.md

```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
```

Comment:
I think you should change this title line to be more specific, e.g. "Ryan's STEP Portfolio", or just remove this line and the following blank line.